FSW/src/util/commandHandling.cpp

```diff
@@ -269,15 +267,20 @@ void executeCommand(int command) {
             Serial.println("Command Received - ADCS_POINTING_AT_SUN set to false.");
             break;
 
-        // Downlink
+        // Memory
         case DOWNLINK_START:
-            scienceMode.downlinkEvent.invoke();
+            downlinkEvent.invoke();
             Serial.println("Command Received - Downlink will begin when payload enters standby.");
             break;
 
+        case SCRUB_FLASH:
+            scrubEvent.invoke();
+            Serial.println("Command Received - Scrub initiated.");
```

Comment:
Maybe a touch more explicit: "Memory Scrub Initiated."

Reply:
Changed to "Flash scrub initiated."